src/Pages/ProductList/Components/SideFilterBar/Components/ColorFilterBox/ColorFilterBox.js

```diff
@@ -0,0 +1,71 @@
+import React from "react";
+import { FiMinus } from "react-icons/fi";
+import { FiPlus } from "react-icons/fi";
+import { colorValues } from "../../../../../../config";
+import "./ColorFilterBox.scss";
+
+class ColorFilterBox extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      fold: false,
+      colorFilterValue: "",
+    };
+  }
+
+  handleFolder = () => {
+    this.setState({ fold: !this.state.fold });
+  };
+
+  handleFilterChange = (e) => {
+    this.props.handleFilterChange(e.target.name, e.target.value);
+    this.setState({ colorFilterNumber: e.target.value });
+  };
+
+  handleClassName = (filter) => {
+    const { colorFilterValue } = this.state;
+    if (colorFilterValue === filter) {
+      if (filter === "white") {
+        return "white selected";
+      }
+      return "selected";
+    } else if (filter === "white" && colorFilterValue !== filter) {
+      return "white";
+    }
```

Comment:
```suggestion
    if (colorFilterValue === filter) {
      if (filter === "white") {
        return "white selected";
      }
      return "selected";
    } else if (filter === "white" && colorFilterValue !== filter) {
      return "white";
    }
```
조건 걸어준 모양이 조금 어색해 보입니다.
1. 1depth 에서 colorFilterValue가 filter와 같은지 여부를 가지고 분기를 나눕니다.
2. 그런데 자세히 살펴보면 사실 filter === "white"가 공통 조건이고 colorFilterValue가 filter와 같냐 같지 않냐를 가지고 분기가 랍생하게 됩니다.

Reply:
수정 완료하였습니다:)